crates/l2/sequencer/l1_committer.rs

```diff
@@ -85,9 +85,11 @@ impl Committer {
     }
 
     async fn main_logic(&mut self) -> Result<(), CommitterError> {
-        let block_number =
-            EthClient::get_next_block_to_commit(&self.eth_client, self.on_chain_proposer_address)
-                .await?;
+        let block_number = 1 + EthClient::get_last_committed_block(
```

Comment:
This`1` is because we skip the genesis?

Reply:
That's because we want to send the next one to the last committed. `next_block_to_commit == last_commited_block + 1`